Week1/Homework/js-exercises/compareArrays.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+var me = ["Abuzer", 29, "unemployed", "bachelor graduate"];
+
+var tesla = [
+  "model S",
+  "model X",
+  "model Y",
+  "model 3",
+  "Roadster",
+  "Cyber Truck",
+  "Solar Panel"
+];
+
+console.log("length of the array me is ", me.length);
+console.log("length of the array tesla is ", tesla.length);
+
+if (me.length == tesla.length) {
```

Comment:
Ah, I think this is almost correct, in Javascript there is a difference between `==` and `===`  See [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators)

Reply:
Yeah I know. == checks the value  and === checks the value and the type. When I used length attribute or method, the result will be an integer for that reason I wrote ==. But I will change it.

Reply:
Yeah I know. == checks the value  and === checks the value and the type. When I used length attribute or method, the result will be an integer for that reason I wrote ==. But I will change it.